scheduler/task.go

```diff
@@ -42,6 +42,57 @@ func createTaskInfo(task types.EremeticTask, offer *mesos.Offer) (types.Eremetic
 		})
 	}
 
+	var portResources []*mesos.Value_Range
+	var portMapping   []*mesos.ContainerInfo_DockerInfo_PortMapping
+
+	if(len(task.Ports) > 0) {
```

Comment:
Hmm. that's true. However reconstructing that into two ranges by finding sequences should be doable. There's also the really simple method of mapping that into 5 1-length ranges like (3001-3001, 3002-3002, ...) which I think would still be legal?